File: APICuponSmart/src/java/modelo/pojo/respuesta/RespuestaCiudad.java
package modelo.pojo.respuesta;

import java.util.List;
import modelo.pojo.entidad.Ciudad;

public class RespuestaCiudad extends Mensaje{
    private List<Ciudad> contenido;
    
    public List<Ciudad> getContenido(){
        return contenido;
    }
    
    public void setContenido(List<Ciudad> contenido){
        this.contenido = contenido;
    }
}

File: AppCuponSmart/app/src/main/java/uv/tc/appcuponsmart/data/model/respuesta/RespuestaCiudad.kt
package uv.tc.appcuponsmart.data.model.respuesta

import uv.tc.appcuponsmart.data.model.entidad.Ciudad

class RespuestaCiudad(
    error: Boolean,
    mensaje: String,
    var contenido: MutableList<Ciudad>? = null
): Mensaje(error, mensaje)